docs/_docs/uicollectionviewinterop.md

```diff
@@ -71,8 +71,12 @@ For this example, the data source method `collectionNode:nodeBlockForItemAtIndex
   </pre>
 
   <pre lang="swift" class = "swiftCode hidden">
-  // Click the "Edit on GitHub" button at the bottom of this 
-  // page to contribute the swift code for this section. Thanks!
```

Comment:
please do not remove the existing text; please add your example before this block. Thanks.